assets/test/specs/fixtures/enhanced-tables.html

```diff
@@ -0,0 +1,87 @@
+
```

Comment:
we have a pattern of appending `-fixture` to the fixture filename can you add this like the others in the fixtures folder